kiosk/src/Components/ScanQR.tsx

```diff
@@ -85,6 +108,7 @@ const ScanQR: React.FC<IProps> = ({ kiosk }) => {
                         }
```

Comment:
Should `Incorrect format for a share link' have different styling to distinguish it from the 'Submit share link'? I'm not sure exactly how it should look but it seems odd for them to be the same. Maybe the font should be smaller or a different color?

Reply:
That's a good call out, thank you. I'm thinking maybe changing the font to that other one that is used for the link currently and making it a bit smaller.